### PR TITLE
ref(css): moved styles out of bootstrap

### PR DESCRIPTION
**Pull request by robinrendle**
Still refactoring Bootstrap so our default styles makes sense. In this PR:

- Refactored `button.less` and `well.less`: deleted unused styles
- Moved everything out of `scaffolding.less` into `base.less`

### CODE REVIEW

File: src/sentry/static/sentry/less/base.less [hunk 1]
```diff
@@ -582,3 +582,52 @@ td,
...
+  &:active,
+  &:focus {
+    position: static;
+    width: auto;
+    height: auto;
+    margin: 0;
+    overflow: visible;
+    clip: auto;
+  }
+}
```
Comment (robinrendle) [inline src/sentry/static/sentry/less/base.less:633]: All of this stuff comes from `scaffolding.less`

**Review by scttcper — APPROVED** [review]

File: src/sentry/static/sentry/less/base.less [hunk 2]
```diff
@@ -582,3 +582,52 @@ td,
...
+// Fixes "clickability" issue (and more generally, the firing of events such as focus as well)
+// for traditionally non-focusable elements with role="button"
+// see https://developer.mozilla.org/en-US/docs/Web/Events/click#Safari_Mobile
+
+[role='button'] {
+  cursor: pointer;
+}
+
+// Horizontal rules
+hr {
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/less/base.less:597]:
```suggestion
// Horizontal rules

hr {
```

**Comment by priscilawebdev** [comment]
Some buttons look different. Shouldn't we look into that?

**Before:**

image


**After:**

image

File: src/sentry/static/sentry/less/base.less (hunk 2 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/less/base.less:597]: should we keep it consistent? 🤷‍♀️

**Pull request closed without merging**